tools/ci_connector_ops/ci_connector_ops/qa_engine/outputs.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import pandas as pd
+
+from .models import ConnectorQAReport
+
+def persist_qa_report(qa_report: pd.DataFrame, path: str, public_fields_only: bool =True):
```

Comment:
This is not called as I disabled GCS persistence for safety.

Reply:
👍🏻 I assume we'll usually persist with all of the info, but can use the public-only if we plan to use it in the public repo anywhere